tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -979,7 +979,7 @@ class MklConvOp : public OpKernel {
 
       // Check if need reorder
       if (add_mkl_shape == output_mkl_shape) {
-        CHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
+        DCHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
```

Comment:
DCHECK gets optimized out when TF is not in debug mode. Any statements inside DCHECK will not be executed in opt mode. It looks like you need `CopyFrom` to be executed so you should call `CopyFrom` in one line, save the result, and `DCHECK` the result in the next line.

If `CopyFrom` doesn't actually need to be run, please let me know and I'll approve the code.